tests/MQEstimator/test_mq.py

```diff
@@ -0,0 +1,168 @@
+from sage.all_cmdline import *
+import pytest
+from math import log2
+from cryptographic_estimators.MQEstimator import (
+    MQProblem,
+    MQEstimator,
+    MQProblem,
+    Bjorklund,
+    BooleanSolveFXL,
+    CGMTA,
+    Crossbred,
+    DinurFirst,
+    DinurSecond,
+    ExhaustiveSearch,
+    F5,
+    HybridF5,
+    KPG,
+    Lokshtanov,
+    MHT,
+)
+from tests.MQEstimator.legacy_implementations.mpkc.algorithms import (
+    Bjorklund as TestBjorklund,
+    BooleanSolveFXL as TestBooleanSolveFXL,
+    CGMTA as TestCGMTA,
+    Crossbred as TestCrossbred,
+    DinurFirst as TestDinurFirst,
+    DinurSecond as TestDinurSecond,
+    ExhaustiveSearch as TestExhaustiveSearch,
+    F5 as TestF5,
+    HybridF5 as TestHybridF5,
+    KPG as TestKPG,
+    Lokshtanov as TestLokshtanov,
+    MHT as TestMHT,
+)
+
+
+ranges = 0.01
+
+test_sets = [
+    # n   m   q
+    [50, 50, 2],
+    [70, 70, 4],
+    [50, 70, 8],
+    [120, 40, 8],
+]
+
+algos = [
+    Bjorklund,
+    BooleanSolveFXL,
+    CGMTA,
+    Crossbred,
+    DinurFirst,
+    DinurSecond,
+    ExhaustiveSearch,
+    F5,
+    HybridF5,
+    KPG,
+    Lokshtanov,
+    MHT,
+]
+
+test_algos = [
+    TestBjorklund,
+    TestBooleanSolveFXL,
+    TestCGMTA,
+    TestCrossbred,
+    TestDinurFirst,
+    TestDinurSecond,
+    TestExhaustiveSearch,
+    TestF5,
+    TestHybridF5,
+    TestKPG,
+    TestLokshtanov,
+    TestMHT,
+]
+
+
+def test_estimates_with_bjorklund():
+    excluded_algorithms = [
+        DinurFirst,
+        DinurSecond,
+        BooleanSolveFXL,
+        HybridF5,
+        Lokshtanov,
+        Crossbred,
+        F5,
+        KPG,
+        CGMTA,
+        MHT,
+    ]
+
+    mq_estimator = MQEstimator(100, 50, 2, excluded_algorithms=excluded_algorithms)
+    result = mq_estimator.estimate()
+    expected_result = {
+        "Bjorklund": {
```

Comment:
Isn't it better to store those results in a file `results.json`? This would make the update process much easier, if we want to add more tests for example.